frontend/Login/LargeOption.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Text, View, StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  bigContainer: {
+    backgroundColor: "white",
+    height: 210,
+    width: 150,
+    borderRadius: 40,
+    margin: 20,
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    boxShadow: "5px 5px 15px 2px rgba(161,161,161,0.39)",
```

Comment:
When I run this branch, I get an error saying that boxShadow is not a valid style property. I think it doesn't exist for ios. See here for shadow on a container: 

shadow: {
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 3,
    },
    shadowOpacity: 0.05,
    shadowRadius: 4.65,
    elevation: 5,
  },

Reply:
Nice catch! I dug into a bit more and it looked like Android and ios have different css properties for box shadows that could make the css a bit messy, so I just deleted for now. I can add them back later if you want though, I just won't be able to test if the ios styles actually work.
![Screen Shot 2022-01-19 at 22 07 38](https://user-images.githubusercontent.com/32073638/150283229-6217041c-fd2b-4b7e-8cce-54ae52ef640b.png)